### DISCUSSION TITLE
Now install zigbee2mqtt on HassOS add on, I want to move all zigbee deivce to zigbee2mqtt installed on raspberry pi 4 linux without new pairing.

### DISCUSSION THREAD

**Discussion by mitrejit** [Koenkk/zigbee2mqtt#11143] [Devices]
Now install zigbee2mqtt on HassOS add on, I want to move all zigbee deivce to zigbee2mqtt installed on raspberry pi 4 linux without new pairing.

**Comment by francisp2** [comment]
Just copy your existing data directory to your new installation

**Reply by netweaver1970** [reply]
And what about converting the configuration.yaml to a jason version?
